sdc/datatypes/hpat_pandas_series_rolling_functions.py

```diff
@@ -305,6 +305,32 @@ def apply_minp(arr, ddof, minp):
     gen_hpat_pandas_series_rolling_ddof_impl(arr_var))
 
 
+@sdc_register_jitable
+def pop_count(value, nfinite, result):
+    """Calculate the window sum without old value."""
+    if not numpy.isnan(value):
+        nfinite -= 1
```

Comment:
I don't think we need to do something with `nfinite`. Could you please remove it?